extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementCard.java

```diff
@@ -61,7 +61,7 @@ public class ContractAgreementCard {
     private AssetDto asset;
 
     @Schema(description = "Contract Policy", requiredMode = Schema.RequiredMode.REQUIRED)
-    private PolicyDto contractPolicy;
+    private UiPolicyDto contractPolicy;
```

Comment:
Why is it named contractPolicy (instead of Policy)? if it was a UiPolicyDto (consistenty with the name of PolicyDefinition)

Reply:
because contract definitions have two policies, the contract policy and the access policy. this helps to avoid the confusion